src/components/course/course-header/CourseRunCardStatus.jsx

```diff
@@ -28,25 +29,33 @@ const CourseRunCardStatus = ({
   isUserEnrolled,
   userCanRequestSubsidyForCourse,
 }) => {
+  const { data: { customerAgreement } } = useSubscriptions();
   const missingUserSubsidyReasonType = missingUserSubsidyReason?.reason;
   const missingUserSubsidyReasonUserMessage = missingUserSubsidyReason?.userMessage;
   const missingUserSubsidyReasonActions = missingUserSubsidyReason?.actions;
-
```

Comment:
NIT: Could we restore the empty line here that was previously in place?

Reply:
It was unnecessary in the code so I removed it.